components/BestPostsSection/BestPostsSection.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from "react";
+import styles from "./BestPostsSection.module.scss";
+import BestPostCard from "../BestPostCard/BestPostCard";
+import { getPost } from "../api";
+
+// export const getStaticProps = async () => {
+//   const BestPostsData = await getPost({
+//     page: 1,
+//     pageSize: 3,
+//     orderBy: "like",
+//   });
+//   const bestPosts = BestPostsData.list;
+
+//   return {
+//     props: {
+//       bestPosts,
+//     },
+//   };
+// };
+
+const BestPostsSection = () => {
+  const [bestPosts, setBestPosts] = useState([]);
+  const [windowWidth, setWindowWidth] = useState(0);
+  const [pageSize, setPageSize] = useState(3);
+
+  const getPageSize = () => {
+    if (windowWidth > 1199) {
+      setPageSize(3);
+    }
+    if (windowWidth < 1199 && windowWidth > 744) {
+      setPageSize(2);
+    }
+    if (windowWidth < 744) {
+      setPageSize(1);
+    }
+  };
+
+  const handleLoad = async () => {
+    const BestPostsData = await getPost({
```

Comment:
```suggestion
    const bestPostsData = await getPost({
```
처음에 대문자를 주는 경우(파스칼케이스)는 보통 타입을 주거나 컴포넌트를 정의할 때 사용됩니다!